components/automate-deployment/pkg/assets/assets.go

```diff
@@ -1,6 +1,6 @@
 package assets
 
-const ChefAutomateUpgradeFromV1SeflTestCRL = `-----BEGIN X509 CRL-----
+const ChefAutomateUpgradeFromV1SelfTestCRL = `-----BEGIN X509 CRL-----
```

Comment:
@stevendanna I could not find where constants in this file were referenced; is this OK to change?

Reply:
I think it is OK.  The other constants are referenced, this one isn't but I still wanted to keep the CRL around in case we wanted it in the future.

Reply:
components/automate-deployment/pkg/a1stub/server.go